.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
you should not add the sittings file to your branch 
I assume that you used `git add . ` which is a bad practice 
to fix it you need to delete this file locally and add your changes (the deleted file) to this branch